### DISCUSSION TITLE
CSV import issues [Jan 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5039] [Discord Threads]
> 🙋‍♂️ **unicornYorkie**  
## What is wrong with CSV Imports  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1054971036619251783/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### 1. please provide the project info   
#### 2. why did you map multiple columns to id?  
#  
> 🙋‍♂️ **unicornYorkie**  
#### I keep getting the ID error so I thought it was a formatting issue. I left Vendor ID “as-is,” and created a new column “ID” as an integer.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1054971036917043310)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
